proto/ibc/core/channel/v1/tx.proto

```diff
@@ -375,4 +380,8 @@ message MsgChannelUpgradeCancel {
 }
 
 // MsgChannelUpgradeCancelResponse defines the MsgChannelUpgradeCancel response type
-message MsgChannelUpgradeCancelResponse {}
+message MsgChannelUpgradeCancelResponse {
+  option (gogoproto.goproto_getters) = false;
+
+  ResponseResultType result = 1;
```

Comment:
It's a bit confusing to me that the result type `Success` means the handshake wasn't aborted or it was aborted, depending on the message type. I wonder if it'd be simpler to add a bool field for message types where the handshake could be aborted

Reply:
We should discuss in engineering since this is part of the API and would be nice to get it right. I can see the point raised and how it might lead to some confusion (but also like the idea of going with `result` types everywhere if at least one handler uses it).